taskfile.yaml

```diff
@@ -103,6 +105,10 @@ tasks:
       - "rm -rf '{{.G_WEBUI_SRC_DIR}}/server/node_modules'"
       - "rm -rf '{{.G_WEBUI_SRC_DIR}}/yscope-log-viewer/node_modules'"
 
+  clean-rust:
```

Comment:
i don't remember why we placed the `clp-json-pkg-tar` and `clp-text-pkg-tar` in between the "clean" tasks though likely we shouldn't. still let's place this directly above `clean-webui` (to also match the proposed order in the `clean` task), so the diff may look nicer when we feel itchy to clean up the order of the pkg tasks